docs/installation.md

```diff
@@ -8,7 +8,6 @@ This document begins with a number of sections concerning native installation:
 * [Configuration](#configuration)
 * [Startup](#startup)
 * [Post-installation sanity check](#post-installation-sanity-check)
-* [What's next](#whats-next)
 

```

Comment:
This is installation guide for the backend. The "What's next" section pointed at the installation of the GUI. I see no point to point at that. Under Zonemaster we have the following text that bind the different parts together. There is no point of extending the installation guide for the backend with other information.

> In order to install and run the different components, have a look at the installation documentation under each of the repositories below:
>
>    zonemaster-engine - which contains the test framework.
    zonemaster-cli - a Command Line Interface for the engine.
    zonemaster-backend - which interfaces between the GUI and an API with the engine.
    zonemaster-gui - A graphical user interface for the engine.